src/data/vehicles.ts

```diff
@@ -0,0 +1,29 @@
+import { csvLoader } from '../shared/csv.macro'
+
+// see src/data/vehicles.csv structure
+type Vehicle = {
+  tracker_id: string
+  organization: string
+  board: string
+  vehicle_type: string
+  model: string
+  door_count: number
+  release_date: string
+  articulated: string // TODO make it bool
+  accessibility: string //TODO make it bool
```

Comment:
@iamandrewluca, is there a way to transform the string `"yes"` into a boolean variable at load stage?

Or should I read the raw data, then iterate over it and produce another structure, using simple if/else logic?

Reply:
We can add a normalization step, before storing data in state. TypeScript `type` should reflect normalized state, instead of state received.